app/src/main/res/values-land/dimens.xml

```diff
@@ -392,7 +392,11 @@
 
   <!-- AudioLanguageFragment -->
   <dimen name="audio_language_recycler_view_padding_bottom">96dp</dimen>
-  
+
+  <!-- ProfileListFragment -->
+  <dimen name="profile_list_fragment_recycler_view_padding_bottom">40dp</dimen>
+
   <!-- StoryFragment -->
```

Comment:
The `profile_list_fragment_recycler_view_padding_bottom` for the landscape xml file is 108dp, please verify this at your end.